### DISCUSSION TITLE
Feature request: Go to last executed cell

### DISCUSSION THREAD

**Discussion by andreaschiappacasse** [microsoft/vscode-jupyter#9572] [New Ideas]
Currently there is a useful button to `Go to Running Cell`, which disappears as soon as the notebook stops running.

I think it would be great to also have a `Go to Last Executed Cell` which could allow the slider to jump to the last executed cell even after it finished running.

My typical use case would be:

- Have a notebook with some kind of script
- Run all 
- Come back after some time to find an error occurred somewhere
- Quickly check where the error occurred by pressing  `Go to Last Executed Cell`

or

- Have a notebook with multiple cells
- Run a computation-heavy cell which takes some time to finish
- Do dome work somewhere else in the notebook
- See from the status bar that the previously running cell finished
- Quickly jump to the output by pressing  `Go to Last Executed Cell`

I think that it would be very natural UI/UX-wise, since we are already used to the similar `Go to Running Cell` functionality in the toolbar.
What do you think about it? Was it ever considered elsewhere?

**Comment by andreaschiappacasse** [comment]
Does this belong here (in vscode-jupyter) or should I create a issue in core? I am not sure which code base would be impacted.

**Reply by greazer** [reply]
Thanks for the feedback @andreaschiappacasse. This was already requested by me. :) Please go to https://github.com/microsoft/vscode/issues/142898 and upvote it and add any further comments you may have.

**Reply by andreaschiappacasse** [reply]
> Thanks for the feedback @andreaschiappacasse. This was already requested by me. :) Please go to [microsoft/vscode#142898](https://github.com/microsoft/vscode/issues/142898) and upvote it and add any further comments you may have.

Perfect! Sorry for the duplicate, I missed it on the main repo.. I commented there with the other use case I often find myself in. 
Thank you!
